src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenTuples.vb

```diff
@@ -870,7 +870,7 @@ End Module
   .maxstack  2
   IL_0000:  ldc.i4.2
   IL_0001:  ldc.i4.3
-  IL_0002:  call       "Function C.M1(Integer, Integer) As (Integer, Integer)"
+  IL_0002:  call       "Function C.M1(Integer, Integer) As ValueTuple(Of Integer, Integer)"
```

Comment:
>ValueTuple [](http://example.com/codeflow?start=59&length=10)

I am curious why "System." is missing here. Is this expected? For example, it is present on the line below

Reply:
good catch.  logic seems wrong vs c#.  i didn't grab all the things it does.

Reply:
Fixed.